src/interfaces/assistants_web/src/components/Agents/AgentsSidePanel.tsx

```diff
@@ -66,75 +79,74 @@ export const AgentsSidePanel: React.FC<React.PropsWithChildren<{ className?: str
       <div
         className={cn(
           'flex h-full flex-grow flex-col gap-y-8 px-4 py-6',
-          'md:transition-[min-width,max-width]',
-          {
-            'md:min-w-agents-panel-collapsed md:max-w-agents-panel-collapsed':
-              !isAgentsSidePanelOpen,
-            'md:min-w-agents-panel-expanded md:max-w-agents-panel-expanded lg:min-w-agents-panel-expanded-lg lg:max-w-agents-panel-expanded-lg':
-              isAgentsSidePanelOpen,
-          }
+          'md:min-w-agents-panel-expanded md:max-w-agents-panel-expanded lg:min-w-agents-panel-expanded-lg lg:max-w-agents-panel-expanded-lg'
         )}
       >
         <div
           className={cn('flex flex-shrink-0 items-center', {
             'justify-between gap-x-3': isAgentsSidePanelOpen || isMobile,
-            'justify-center': !isAgentsSidePanelOpen && isDesktop,
           })}
         >
-          <Transition
-            show={isAgentsSidePanelOpen || isMobile}
-            as="div"
-            enter="transition-all transform ease-in-out duration-200"
-            enterFrom="-translate-x-full"
-            enterTo="translate-x-0"
-          >
-            <Link href="/" shallow>
-              <div className="mr-3 flex items-baseline">
-                <Logo hasCustomLogo={env.NEXT_PUBLIC_HAS_CUSTOM_LOGO === 'true'} />
-              </div>
-            </Link>
-          </Transition>
+          <Link href="/" shallow>
+            <div className="mr-3 flex items-baseline">
+              <Logo hasCustomLogo={env.NEXT_PUBLIC_HAS_CUSTOM_LOGO === 'true'} />
+            </div>
+          </Link>
 
           <IconButton
             iconName="close-drawer"
             onClick={handleToggleAgentsSidePanel}
-            className={cn('transition delay-100 duration-200 ease-in-out', {
-              'rotate-180 transform text-mushroom-400': isAgentsSidePanelOpen || isMobile,
-            })}
+            className={cn(
+              'flex text-mushroom-950 transition delay-100 duration-200 ease-in-out md:hidden',
+              {
+                'rotate-180 transform': isAgentsSidePanelOpen,
+              }
+            )}
+          />
+        </div>
+
+        <div className="flex flex-shrink-0 flex-col gap-y-4">
+          <Button
+            kind="secondary"
+            className="truncate [&_span]:text-evolved-green-700"
+            startIcon={<Icon name="add" kind="outline" className="text-evolved-green-700" />}
+            label="New chat"
+            onClick={handleNewChat}
+            shallow
```

Comment:
we can and should remove the `shallow`, this concept does not exist for the app router